compiler/src/dmd/backend/cg87.d

```diff
@@ -83,9 +83,12 @@ enum NDPP = 0;       // print out debugging info
 @trusted
 bool NOSAHF() { return I64 || config.fpxmmregs; }     // can't use SAHF instruction
 
-enum CW_roundto0 = 0xFBF;
-enum CW_roundtonearest = 0x3BF;
-
+/** 87 Control Word rounding modes */
+enum CW : ushort
```

Comment:
I don't see why this enum name still needs to be so short considering it only has about 10 uses

Reply:
ControlWord.roundtonearest is a little long and repetitive. The instructions to manipulate it are FLDCW, FSTCW